src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/CCJContentProvider.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff.content;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.models.Claim;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class CCJContentProvider {
+    //TODO Data for default judgmenrt PDF
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1135)